src/main/java/br/com/sbk/sbking/networking/kryonet/KryonetSBKingServer.java

```diff
@@ -146,4 +136,15 @@ public boolean nobodyIsConnected() {
   public void sendYourTableIsTo(String gameName, UUID playerIdentifier) {
     this.sendOneTo(new YourTableIsMessage(gameName), playerIdentifier);
   }
+
+  // public void enterTable(){
+  // Table table = this.sbkingServer.getTable();
+  // if (table != null) {
+  // GameServer gameServer = table.getGameServer();
+  // String gameName =
+  // GameNameFromGameServerIdentifier.identify(gameServer.getClass());
+  // this.sendYourTableIsTo(gameName, identifier);
+  // this.sbkingServer.addSpectator(identifier, table.getId());
+  // }
+  // }
```

Comment:
This commented lines should exist?

Reply:
They will be used in the very near future, when players join tables. I just decided to leave it there for the meantime